services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/DefaultAttributeConverterProvider.java

```diff
@@ -80,7 +77,7 @@
  * Given an input, this will identify a converter that can convert the specific Java type and invoke it. If a converter cannot
  * be found, it will invoke a "parent" converter, which would be expected to be able to convert the value (or throw an exception).
  */
-@SdkInternalApi
+@SdkPublicApi
```

Comment:
I think we should spruce up the javadoc a bit to celebrate this class' new status as 'public'. At a minimum mention (preferably with examples) how to use it in a StaticTableSchema builder and a BeanTableSchema annotation.